### PR TITLE
Hussin 06 two player

### PR DESCRIPTION
**Pull request by HussinSaheb**
The game is now playable by two, with point assigned to each,
fixed a bug when time runs out and another event listener to trigger again
added modals for instructions and leaderboards(saved in local storage).
changed the draw timer to lower for demonstration purposes.

### CODE REVIEW

**Comment by HussinSaheb** [comment]
in terms of the merge conflicts, the way i have it right now - is the correct version that i want to keep as it has no issues anymore

File: index.html [hunk 1]
```diff
@@ -10,11 +10,56 @@
     <title>Scribble.IO</title>
   </head>
   <body>
-    <section class="container-fluid">
+    <section class="container">
+        <h1>Welcome to Skribble.io</h1>
       <section class="mainMenu">
```
Comment (AEJackson) [inline index.html]: check indentation

**Review by AEJackson — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (HussinSaheb) [reply index.html]: Hi Anthony, I've changed it. could you please have a look again

**Review by AEJackson — DISMISSED** [review]

File: index.html [hunk 2]
```diff
@@ -10,11 +10,56 @@
     <title>Scribble.IO</title>
   </head>
   <body>
-    <section class="container-fluid">
+    <section class="container">
+      <h1>Welcome to Skribble.io</h1>
       <section class="mainMenu">
```
Comment (AEJackson) [inline index.html:15]: where is the closing tag for this <section>

File: index.html [hunk 3]
```diff
@@ -10,11 +10,56 @@
...
+                </section>
+              </div>
+              <div class="modal-footer">
+                <button type="button" class="btn btn-default" data-dismiss="modal">Close</button>
+              </div>
+            </div>
+          </div>
+        </div>
+        <!-- Button trigger modal -->
+      <section class="play">
```
Comment (AEJackson) [inline index.html]: check indentation

File: js/index.js [hunk 4]
```diff
@@ -1,15 +1,90 @@
...
-  function startGame() {
-    askPlayerAmount();
-    displayGuesses();
-    displayWord();
-    drawCanvas();
+  function startGame(Leaderboard) {
+    var player;
+    switch (playCount) {
+      case 1:
+      player = Leaderboard.getScores()[1];
```
Comment (AEJackson) [inline js/index.js]: indent

File: js/index.js [hunk 5]
```diff
@@ -1,15 +1,90 @@
...
+      $("#PlayerTurn").text("Player two's turn to guess");
+      // call game function with the player object passed as argument.
+      game(player);
+      break;
+      case 2:
+      player = Leaderboard.getScores()[0];
```
Comment (AEJackson) [inline js/index.js]: indent

File: js/index.js [hunk 6]
```diff
@@ -1,15 +1,90 @@
...
+      $("#PlayerTurn").text("Player one's turn to guess")
+      game(player);
+      break;
+      case 3:
+      player = Leaderboard.getScores()[1];
```
Comment (AEJackson) [inline js/index.js]: indent

File: js/index.js [hunk 7]
```diff
@@ -1,15 +1,90 @@
...
+      $("#PlayerTurn").text("Player two's turn to guess")
+      game(player);
+      break;
+      case 4:
+      player = Leaderboard.getScores()[0];
```
Comment (AEJackson) [inline js/index.js]: indent

**Review by AEJackson — CHANGES_REQUESTED** [review]

**Review by AEJackson — APPROVED** [review]

**Pull request merged**